src/main/java/service/impl/TransactionProcessorImpl.java

```diff
@@ -0,0 +1,19 @@
+package service.impl;
+
+import java.util.List;
+import model.Transaction;
+import service.TransactionProcessor;
+import strategy.OperationHandler;
+import strategy.OperationStrategy;
+
+public class TransactionProcessorImpl implements TransactionProcessor {
+    @Override
+    public void createReport(List<Transaction> transactionsList,
+                             OperationStrategy operationStrategy) {

```

Comment:
Don’t pass OperationStrategy to the method better avoid dependencies between services (think  how can you do it), but if it is impossible create a class level field and initialize it in the constructor